src/renderer/settings/general-settings-component.ts

```diff
@@ -525,6 +525,22 @@ export const generalSettingsComponent = Vue.extend({
                             </div>
                         </div>
 
+                        <div class="settings__option">
+                            <div class="settings__option-name">{{ translations.generalSettingsDecimalSeparator }}</div>
+                            <div class="settings__option-content">
+                                <div class="field is-grouped is-grouped-right">
+                                    <div class="buttons has-addons">
+                                        <button class="button"
+                                            :class="{ 'is-success': config.generalOptions.decimalSeparator == '.' }"
+                                            @click="config.generalOptions.decimalSeparator = '.'; updateConfig();">.</button>
+                                        <button class="button"
+                                            :class="{ 'is-success': config.generalOptions.decimalSeparator == ',' }"
+                                            @click="config.generalOptions.decimalSeparator = ','; updateConfig();">,</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
```

Comment:
Is there a reason why you decided not to add a normal `select` with the options "," and "."?

Reply:
I oriented on [Bootstrap radio buttons](https://getbootstrap.com/docs/4.0/components/buttons/#checkbox-and-radio-buttons) because of two advantages compared with a select: You see all options at once, without the need to open the dropdown. And you can change the setting with just one click. Sadly, Bulma does not support this natively, but the result is pretty much the same.

Reply:
Fair enough.